manifests/federation/0.0.5/federation.v0.0.5.clusterserviceversion.yaml

```diff
@@ -23,23 +23,123 @@ spec:
     quickly get up and running with this powerful concept. Federation is a key
     part of any Hybrid Cloud capability.
 
-    This CSV deploys Federation scoped to the namespace it is deployed into.
-    Note that in this configuration, Federation cannot handle cluster-scoped
-    resources. Use this CSV if you want to add federation capability to a single
-    namespace.
+    Kubernetes Federation is currently in preview. This operator will be updated
+    as we release new versions of the [upstream repository](https://github.com/kubernetes-sigs/federation-v2).
+
+    TODO: note about uninstalling old operator before upgrading
+
+    ## Using Federation
 
-    ### Using Federation
     After configuring and starting your Federation instance, use the command
     line tool to enable each type of object you would like federated to all of
     your clusters. This deploys Federation in a namespace-scoped configuration.
-    This means that only objects that share the Namespace where your Federation
-    instance is deployed can be federated. You must enable Federation support
-    for Namespaces in this configuration before other types will work. Use
-    `kubefed2 enable namespaces`.
-
-    Next, use `kubefed2 enable <API type>` for the other objects you would like to federate.
-    Finally, [register the other clusters](https://github.com/kubernetes-sigs/federation-v2/blob/master/docs/userguide.md#join-clusters) that will be federated to.
-    See the [full user guide](https://github.com/kubernetes-sigs/federation-v2/blob/master/docs/userguide.md) for other capabilities of Federation.
+    This means that only objects in the namespace where federation is deployed
+    can be federated.
```

Comment:
One part I struggled at first was that I assumed I had to install the federation operator in all clusters... After reading this new version I finally figured out that this is not required. Still, I'd add something like the following, in case more people make the same mistake:

> Also, the federation operator needs to be deployed into only one of the clusters since it will remotely access all joined clusters via their APIs, which also need be enabled per resource type for the sync to succeed.

Reply:
That is a really good suggestion, thanks!